src/packagedcode/__init__.py

```diff
@@ -97,6 +98,7 @@
     debian.DebianSourcePackageTarballHandler,
 
     distro.EtcOsReleaseHandler,
+    dockerfileLABEL.DockerfileHandler,
```

Comment:
```suggestion
    dockerfile.DockerfileHandler,
```
What about this, we usually don't use uppercase here in file/module names and maybe this name is better?